packages/redux-routine/package.json

```diff
@@ -26,11 +26,13 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"types": "build-types",
 	"sideEffects": false,
 	"dependencies": {
 		"@babel/runtime": "^7.13.10",
 		"is-promise": "^4.0.0",
 		"lodash": "^4.17.21",
+		"redux": "^4.1.0",
```

Comment:
This is a bit of a red flag. We're changing the package's declared dependencies to add types 🤔 

Redux bundles its types, so depending on just types isn't an option 😕 

Why do we use the Redux types? The interfaces we use are structurally aligned with Redux types, but we don't actually depend on redux in our runtime.

Introducing a direct dependency tends to create headaches for consumers because suddenly versions don't align, multiple copies start appearing in `node_modules`, etc.

I'd discourage adding this dependency.

`AnyAction` is a trivial type, we could duplicate it here. [The `Middleware` type, however, is more complex and depends on more parts of `redux`](https://github.com/reduxjs/redux/blob/8b94bcfd9ef84e13a1cb6f2bbee1fc8838ad82ae/src/types/middleware.ts#L22-L30).

It may be better to use a `peerDependency` in this case. We expect `redux` to be included in a project but we won't pull it in ourselves.

@jsnajdr will you check my reasoning here?

Reply:
A `peerDependency` is best. `redux-routine`, as a Redux middleware, is essentially a Redux plugin.

`redux-thunk` declares `redux` as a `devDependency` which also works. The `index.d.ts` files are only used during development, after all.

Reply:
Fixed in 38d8017